### ISSUE TITLE
The metric of the pretrained model is not match the paper.

### ISSUE THREAD

**Issue by GuideWsp** [zhaohengyuan1/PAN#1]
I test the pretrained model with your code, but the metric is not match with the paper, the gap is very big(nearly 2dB).
Hoping for your reply.

**Comment by zhaohengyuan1** [comment]
"The widely used peak signal to noise ratio (PSNR) and the structural similarity index (SSIM) on the Y channel are
used as the evaluation metrics."

**Comment by GuideWsp** [comment]
The "test.py" can calculate the psnr on rgb and ycbcr, but both metric can't match with the paper.
The "test_demo.ipynb"'s metric is not match the test_PANx4_DF2Ktrain_200912-232209.log
The set5 of "test_demo.ipynb" is about29.37dB, while the test_PANx4_DF2Ktrain_200912-232209.log's metric is 32.13.
All the result are from your code. I don't know why? If possible, you can check why.

**Comment by GuideWsp** [comment]
![image](https://user-images.githubusercontent.com/9530892/95696077-4cd59880-0c6c-11eb-9d71-ac85b3d8262f.png)
This is the results tested by your code.
The result is same as "test_demo.ipynb", but not match your log.

**Comment by caokaibin** [comment]
I had the same issue. I also retrained the model but the performance is still far below what was reported in the paper.

**Comment by GuideWsp** [comment]
> I had the same issue. I also retrained the model but the performance is still far below what was reported in the paper.

I had used the official pretrained model to test its metrics, they can not match the paper, the gap is so big.
I also train it from the scratch, but the metric is nearly the provided model, still far below the paper.

**Comment by zhaohengyuan1** [comment]
I'm sorry about this. The reason is that I added a parameter of pytorch function F.interpolate(x,...,**align_corners=True**) when i uploaded my code to github without any evaluation. The align_corners should be False like in file SRResNet_arch.py. You can edit it to False. The code has been updated. Thanks for your reminder~ If you have any questions, please email me hy.zhao1@siat.ac.cn or wechat: zzzhy0331.

**Comment by GuideWsp** [comment]
Thanks for your reply, everything is OK!
